src/vdaf/prio3.rs

```diff
@@ -1111,7 +1154,13 @@ where
     ) -> Result<PrepareTransition<Self, SEED_SIZE, 16>, VdafError> {
         if self.typ.joint_rand_len() > 0 {
             // Check that the joint randomness was correct.
-            if step.joint_rand_seed.as_ref().unwrap() != msg.joint_rand_seed.as_ref().unwrap() {
+            if (!step
+                .joint_rand_seed
+                .as_ref()
+                .unwrap()
+                .ct_eq(msg.joint_rand_seed.as_ref().unwrap()))
```

Comment:
FYI there's a provided `ct_ne()` method, which will let us drop the separate `!`.

I think sidechannels from this comparison are less sensitive because it always passes if the client and aggregators are honest, all aggregators know the joint randomness seed if the client is honest, and the client's report shares determine both sides of the comparison. We may as well keep it constant time for peace of mind.